examples/auth.py

```diff
@@ -0,0 +1,70 @@
+import os
+import json
+from locust import HttpUser, events
+from flask import url_for, Blueprint, request, make_response, redirect, session
+from flask_login import UserMixin, login_user
+
+
+class LocustHttpUser(HttpUser):
+    pass
+
+
+class User(UserMixin):
```

Comment:
Can this class be named something else? To avoid confusion with locusts own User class.